### PR TITLE
#Feature-8 Added-Card-Component

### PR DESCRIPTION
**Pull request by sushmita12345**
Card component added to component library.

Sample
Screenshot 2022-02-02 at 9 25 05 PM

### CODE REVIEW

File: assets/css/card.css [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+    border: .1rem solid var( --black); 
+}
+
+.Card-container-shadow{
+    width: 40%;
+    display: flex;
+    flex-direction: column;
+    border-radius: .5rem;
+    background-color: var(--white);
+    box-shadow: rgba(255, 255, 255, 0.1) 0px 1px 1px 0px inset, rgba(50, 50, 93, 0.25) 0px 50px 100px -20px, rgba(0, 0, 0, 0.3) 0px 30px 60px -30px;
```
Comment (PuneetP16) [inline assets/css/card.css:22]: Add variable instead of direct colours

File: assets/css/card.css [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+    height: 7rem;
+    width: 6.5rem;
+    padding: 3rem;
+    border-radius: 50%;
+}
+
+.card-top-content{
+    padding: 2rem;
+    flex-wrap: wrap;
+} */
```
Comment (PuneetP16) [inline assets/css/card.css:140]: Remove unused commented lines

File: assets/css/card.css [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+/* .card-two{
+    display: flex;
+    padding: 1rem;
+}
+
+.card-bottom-left {
+    display: inline-flex;
+    margin-right: auto;
+    gap: 1rem; 
+} */
```
Comment (PuneetP16) [inline assets/css/card.css:159]: same

**Review by PuneetP16 — APPROVED** [review]
GTM after necessary changes

File: assets/css/card.css (hunk 1 above)
Comment (sushmita12345) [reply assets/css/card.css:22]: Thanks, changes has been done.

File: assets/css/card.css (hunk 2 above)
Comment (sushmita12345) [reply assets/css/card.css:140]: Thanks, changes has been done.

File: assets/css/card.css (hunk 3 above)
Comment (sushmita12345) [reply assets/css/card.css:159]: Thanks, changes has been done.

**Review by palash23jain — APPROVED** [review]

**Pull request merged**